examples/examples.go

```diff
@@ -160,6 +163,67 @@ func CreateAccountWithContracts(flowClient *client.Client, publicKeys []*flow.Ac
 	return account
 }
 
+func FundAccount(flowClient *client.Client, address flow.Address, amount float64) {
+	serviceAcctAddr, serviceAcctKey, serviceSigner := ServiceAccount(flowClient)
+
+	referenceBlockID := GetReferenceBlockId(flowClient)
+
+	fungibleTokenAddress := flow.HexToAddress(conf.Contracts["FungibleToken"])
+	flowTokenAddress := flow.HexToAddress(conf.Contracts["FlowToken"])
+
+	recipient := cadence.NewAddress(address)
+	uintAmount := uint64(amount * sema.Fix64Factor)
+	cadenceAmount := cadence.UFix64(uintAmount)
+
+	fundAccountTx := flow.NewTransaction().
+		SetScript([]byte(fmt.Sprintf(`
+			import FungibleToken from 0x%s
+			import FlowToken from 0x%s
+			
+			transaction(recipient: Address, amount: UFix64) {
+				let tokenAdmin: &FlowToken.Administrator
+				let tokenReceiver: &{FungibleToken.Receiver}
+			
+				prepare(signer: AuthAccount) {
+					self.tokenAdmin = signer
+						.borrow<&FlowToken.Administrator>(from: /storage/flowTokenAdmin)
+						?? panic("Signer is not the token admin")
+			
+					self.tokenReceiver = getAccount(recipient)
+						.getCapability(/public/flowTokenReceiver)
+						.borrow<&{FungibleToken.Receiver}>()
+						?? panic("Unable to borrow receiver reference")
+				}
+			
+				execute {
+					let minter <- self.tokenAdmin.createNewMinter(allowedAmount: amount)
+					let mintedVault <- minter.mintTokens(amount: amount)
+			
+					self.tokenReceiver.deposit(from: <-mintedVault)
+			
+					destroy minter
+				}
+			}
+			`, fungibleTokenAddress, flowTokenAddress))).
+		AddAuthorizer(serviceAcctAddr).
+		AddRawArgument(jsoncdc.MustEncode(recipient)).
+		AddRawArgument(jsoncdc.MustEncode(cadenceAmount))
+	fundAccountTx.
```

Comment:
Is there a particular reason why we have to "re-start" the builder chain here? Can this line be removed?